plugins/kosync.koplugin/main.lua

```diff
@@ -384,6 +383,7 @@ function KOSync:doRegister(username, password)
         custom_url = self.kosync_custom_server,
         service_spec = self.path .. "/api.json"
     }
+    Device:setIgnoreInput(true)
```

Comment:
Might be worth adding a small comment to all of them - to avoid future readers to have to go look at that/why:
`-- on Android to avoid ANR (no-op on other platforms)`